src/offset/local/windows.rs

```diff
@@ -25,7 +25,8 @@ use super::{FixedOffset, Local};
 use crate::{DateTime, Datelike, LocalResult, NaiveDate, NaiveDateTime, NaiveTime, Timelike};
 
 pub(super) fn now() -> DateTime<Local> {
-    tm_to_datetime(Timespec::now().local())
+    let datetime = tm_to_datetime(Timespec::now().local());
+    datetime.single().expect("invalid time")
```

Comment:
An expectation is that the system will not return an invalid time.

Reply:
This looks reasonable, in the case of `now()`